ui/app/models/pki/crl.js

```diff
@@ -28,11 +28,12 @@ export default class PkiCrlModel extends Model {
   @attr('boolean') unifiedCrlOnExistingPaths;
   */
 
-  // edit form ttl attrs
-  @attr('object', {
+  @attr('ttl', {
+    enabledKey: 'auto_rebuild',
+    durationKey: 'auto_rebuild_grace_period',
+    isOppositeValue: false,
     label: 'Auto-rebuild on',
     labelDisabled: 'Auto-rebuild off',
-    editType: 'ttl',
```

Comment:
This would still be required to show the `TtlPicker` in the `FormField` component but it might be better to add a condition to also look for the ttl type. What do you think?
```hbs
{{else if (or (eq @attr.type "ttl") (eq @attr.options.editType "ttl"))}}
```

Reply:
I'm not using the form field component for this model, and this model only appears in one form